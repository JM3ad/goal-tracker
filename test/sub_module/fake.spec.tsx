import React from "react";
import renderer from "react-test-renderer";
import Fake from "src/app/sub_module/fake";
import { render, screen } from "@testing-library/react";

test("Fake component matches the snapshot", () => {
    const component = renderer.create(<Fake />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

test("Fake display contains the word Testing", () => {
    render(<Fake />);
    expect(screen.getByRole("heading").textContent).toEqual("Testing");
});
