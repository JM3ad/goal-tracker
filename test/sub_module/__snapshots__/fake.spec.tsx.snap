// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fake component matches the snapshot 1`] = `
<h2>
  Testing
</h2>
`;
